index.html

```diff
@@ -0,0 +1,162 @@
+    rel="stylesheet" />
+  <link rel="stylesheet" href="style.css" />
+  <!-- <link rel="stylesheet" href="debug.css" /> -->
+  <title>Electronic Event 2024</title>
+</head>
+
+<body>
+  <div class="globe" id="globe">
+    <header>
+      <!-- Top Menu Mobile-->
+      <nav>
+        <div class="top-bar-icon">
+          <a href="#"><img id="hamburger" class="openMenu" src="assets/images/icon-hamburger.svg"
+              alt="Menu Mobile" /></a>
+        </div>
+        <div class="menu mobile-only hidden" id="mobileMenu">
+          <div class="close_btn">
+            <img src="assets/images/icon-cancel.svg" alt="menu cancel" />
+            <div class="top-bar">
+              <div class="top-bar-left">
+                <ul class="mobile-only">
+                  <li><a href="index.html">Language</a></li>
+                  <li><a href="about.html">About</a></li>
+                  <li><a href="#">Events</a></li>
+                  <li><a href="#">Speakers</a></li>
+                  <li><a href="#">Contact</a></li>
+                </ul>
+              </div>
+          </div>
+         </div> 
+        </div> 
+      </nav>
+      <nav class="desktop-only mobile-none">
+        <div class="top-bar">
+          <ul class="sm-menu">
+            <li class="small-menu"><img src="assets/images/facebook.png" alt="logo facebook"></li>
+            <li class="small-menu"><img src="assets/images/twitter.png" alt="logo twitter"></li>
+            <li class="small-menu">English</li>
+            <li class="small-menu">My Page</li>
+            <li class="small-menu">Logout</li>
+          </ul>
+        </div>
+        <div class="bottom-bar">
+          <img src="assets/images/advanced-energy-vector-logo.png" alt="logo" />
+          <ul class="big-menu">
+          <!-- </ul> -->
+          <!-- <ul class="big-menu-right"> -->
+            <li class="big-menu-right">
+              <a href="index.html">Index</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="about.html">About</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="#program">Program</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="#speak">Speakers</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="#sponsors">Sponsors</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="index.html">News</a>
+            </li>
+            <li class="big-menu-right"><button class="contact">Contact Us</button>
+            </li>
+          </ul>
+        </div>
+        </nav>
+    <!-- Top Menu Desktop-->
+    </header>
+    <!-- Main Welcome Text-->
+    <section>
+      <div>
+        <span id="greeting" class="greeting">"Hello! Energy Event!</span>
+        <h1 class="event-name">ENERGY EVENT<br>
+          GLOBALLY<br>
+          2024</h1>
```

Comment:
**Suggested Changes**
- [ ] Currently, your `font-style` looks slanted, unlike in the given design where it is straight. Let us try to get as close as possible to the design.

Reply:
Talked to the previous code reviewer and she accepted the Italic version of the font